src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -15,105 +16,97 @@ import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
 const PrepareBanner = data => {
+  getExtraKnobs(data);
+  if (data.link.link.label) {
+    data.link.link.label = text(
+      'Link label',
+      data.link.link.label,
+      buttonLabels.required
+    );
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.description) {
+    data.description = text(
+      'Description',
+      data.description,
+      buttonLabels.required
+    );
+  }
+
+  if (data.centered) {
+    data.centered = boolean('Centered', data.centered, buttonLabels.optianal);
+  }
+
+  if (data.image) {
+    data.image = text('Image', data.image, buttonLabels.optianal);
```

Comment:
Well, also here you have the same typo as before and the label should be image, not Image